examples/server/server.js

```diff
@@ -10,6 +10,11 @@ const options = {
 const server = mc.createServer(options)
 const mcData = require('minecraft-data')(server.version)
 const loginPacket = mcData.loginPacket
+function chatText (text) {
```

Comment:
I updated this server example since it's the only one that seems to have chat handling been updated for 1.20